integration-tests/contracts/contract_vrf_models.go

```diff
@@ -118,8 +118,7 @@ type VRFBeaconConsumer interface {
 		confirmationDelayArg *big.Int,
 		callbackGasLimit uint32,
 		arguments []byte,
-	) error
-
+	) (*types.Receipt, error)
```

Comment:
Do you need the full receipt here? It seems you're only using it to log the tx hash?

Reply:
`	receipt, err := consumer.RequestRandomnessFulfillment(
		numberOfRandomWordsToRequest,
		subscriptionID,
		confirmationDelay,
		100_000,
		nil,
	)
	require.NoError(t, err)
	log.Info().Interface("TX Hash", receipt.TxHash).Msg("Randomness Fulfillment requested from Consumer contract")

	err = chainClient.WaitForEvents()
	require.NoError(t, err)

	requestID := getRequestId(t, consumer, receipt, confirmationDelay)`